### PR TITLE
Vision copyedit

### PR DESCRIPTION
**Pull request by BasThomas**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Copyedit of the Vision document.

Does this close any currently open issues?
------------------------------------------
…

Any relevant logs, error output, etc?
-------------------------------------

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Devices/Simulators:** …

**iOS Version:** …

**Swift Version:** …

**MessageKit Version:** …

### CODE REVIEW

File: VISION.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Instead, MessageKit will provide you with hooks to easily handle your different
 We will strive to support the 3 latest versions of iOS.
 
 - **Objective-C Compatability**: 
-We will not sacrifice functionality or an idiomatic Swift API to support Objective-C.
+We will not sacrifice functionality or an idiomatic Swift API to support Objective-C, but would love to improve Objective-C Compatability where possible.
 
 - **Layouts**: 
-We will favor programmatic layouts over Xibs where ever possible.
+We will favor programmatic layouts over `.xib`s where ever possible.
```
Comment (BasThomas) [inline VISION.md]: Can we add a reasoning behind this maybe?

File: VISION.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Instead, MessageKit will provide you with hooks to easily handle your different
 We will strive to support the 3 latest versions of iOS.
 
 - **Objective-C Compatability**: 
-We will not sacrifice functionality or an idiomatic Swift API to support Objective-C.
+We will not sacrifice functionality or an idiomatic Swift API to support Objective-C, but would love to improve Objective-C Compatability where possible.
```
Comment (SD10) [inline VISION.md]:
```Swift
"Compatability".lowercased()
``` 
please 😃

File: VISION.md (hunk 1 above)
Comment (SD10) [reply VISION.md]: Yeah, I'm pretty biased about storyboards/xibs. They're slow (parsing XML -> constraints) and prone to merge conflict. I'd rather do the dirty work and provide a better final product to MessageKit users

File: VISION.md (hunk 2 above)
Comment (BasThomas) [reply VISION.md]: Was just trying to be in line with the header though!

**Review by SD10 — APPROVED** [review]

**Comment by SD10** [comment]
I think this is failing because the tests have changed on `master` now that I merged the `v0.10.0` branch. Will just merge this and see what happens 👍

**Pull request merged**